app/serializers/logs_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class LogsSerializer < ApplicationSerializer
+  attributes :id, :item_type, :event, :user_id, :username, :created_at
+
+  belongs_to :encryptable
+
+  def user_id
+    object.whodunnit
+  end
+
+  def username
+    User.find(object.whodunnit).username
```

Comment:
this creates also n+1 queries if user is not included in list query

Reply:
There is no association between Users and Versions therefore I don't think we can use include the username with the Versions query. Is there another way to avoid this n+1 query?

Reply:
I found a way to do it by using join on the user table in the log query :raised_hands:

Reply:
you should probably use *includes* over joins in that case.
see https://medium.com/@swapnilggourshete/rails-includes-vs-joins-9bf3a8ada00